app/code/Magento/Inventory/Model/Source.php

```diff
@@ -0,0 +1,352 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Model\AbstractExtensibleModel;
+use \Magento\InventoryApi\Api\Data\SourceInterface;
+
+/**
+ * Class Source,
+ * provides implementation of the SourceInterface which adds the possibility
+ * for a Merchant to map existing physical sources to some particular sales channels
+ * this model holds the information like name and description of this physical sources.
+ *
+ * @codeCoverageIgnore
+ */
+class Source extends AbstractExtensibleModel implements SourceInterface
```

Comment:
Btw , need to clarify , do we need to have "Code" for Source interface.
Like code in the ERP system